.taskcluster.yml

```diff
@@ -81,11 +82,11 @@ tasks:
               - statuses  # Automatically added by taskcluster-github. It must be explicit because of Chain of Trust
               - $if: 'trust_level == 3'
                 then:
-                  - tc-treeherder.v2.fenix.${head_rev}
+                  - tc-treeherder.v2.${project_name}.${head_rev}
```

Comment:
Is the `fenix` -> `${project_name}` replace here for easier copy-pastability?

Reply:
Yeah, just to make things better when porting the patch to reference-browser. I'm okay reverting it if it's an issue.

Reply:
No worries, I think it's a good pragmatic solution. It's kinda :cry: that we need to write our code for easy copy-pastability rather than easy re-use, but our hands our tied since it's impossible to have re-use in our `.taskcluster.yml`.

:+1: